src/renderer/src/pages/settings/settings-general.tsx

```diff
@@ -79,28 +102,40 @@ export function SettingsGeneral({
         </Button>
       </div>
 
+      <h3>{t("language")}</h3>
+      <>
+        <Select value={form.language} onChange={handleLanguageChange}>
+          {Object.keys(languageResources).map((language) => (
+            <option key={language} value={language}>
+              {ISO6391.getName(language)}
```

Comment:
Could you try with getNativeName instead of getName?

I think it would be better to show the languages names ready in their native language